docs/asset-store/docs/05-01-asset-store.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Asset Store chart
+type: Configuration
+---
+
+To configure the Asset Store chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+To learn more about how to use overrides in Kyma, read [this document](/root/kyma/#configuration-helm-overrides-for-kyma-installation). See also examples of [top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides) and [sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides).
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **minio.mode** | Specifies Minio mode. | `standalone` |
+| **minio.environment.MINIO_BROWSER** | Enables browsing Minio storage. By deafult, the Minio browser is turned off for security reasons. You can change the value to `on` to use the browser. | `"off"` |
```

Comment:
please specify that once you enable browser it is available under `https://minio.{DOMAIN}/minio/`, for example `https://minio.kyma.local/minio/`